search_service/api/table.py

```diff
@@ -161,6 +162,16 @@ def post(self) -> Iterable[Any]:
             msg = 'The search request payload is not available in the request'
             return {'message': msg}, HTTPStatus.BAD_REQUEST
 
+        if 'filters' in search_request:
```

Comment:
is it an issue on ES or both ES and Atlas? If only ES, could we do it on the ES proxy?

Also we need a unit test for this check.